data/campaigns/The_Hammer_of_Thursagan/scenarios/09_The_Underlevels.cfg

```diff
@@ -1294,6 +1294,13 @@
             result=victory
             carryover_report=no
         [/endlevel]
+
+        #ifdef HARD
```

Comment:
I think the checks are complaining about the indentation here - suggest no indent for the `#if` statements and align the WML according to how it should be if there were no `#if` statements.

Reply:
Still not working. I'm somewhat confused here. Is there something about WML indentation I'm missing?

Reply:
This particular instance is resolved.